src/components/SlateEditor/plugins/heading/index.tsx

```diff
@@ -28,7 +28,7 @@ export const headingSerializer: SlateSerializer = {
   deserialize(el: HTMLElement, children: Descendant[]) {
     const tag = el.tagName.toLowerCase();
     if (tag === 'h1') {
-      return jsx('element', { type: TYPE_HEADING, level: 1 }, children);
+      return jsx('element', { type: TYPE_HEADING, level: 2 }, children);
```

Comment:
Dette ser litt rart ut. Det er ikkje mogeleg å sette inn h1 i koden via slate i det heile tatt, men kun via html-editor. Kanskje vi ikkje skal legge oss opp i om en med html-tilgang setter inn h1 i en tekst. Må eventuellt avklares med ndla.

Reply:
Det er implementert på samme måte i master nå:
https://github.com/NDLANO/editorial-frontend/blob/949b059f4de048b603e3e98254b44d91dc8c4bf7/src/util/slateHelpers.jsx#L24

Helt enig i at det er bedre å gi fritt spillerom i html-editoren og mulighet til å legge inn `<h1>` uten at det forsvinner hver gang de åpner vanlig editor.

Reply:
Om det er slik i master så er det nok det dei eventuellt er vant med. Då kan vi rendre det til h2 for h1 og h2, samt til h3 for h3, h4, h5 og h6 som i master.